eng/build.ps1

```diff
@@ -14,25 +14,43 @@ Param(
   [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
 )
 function Print-Usage() {
-    Write-Host "Default if no actions are passed in: -restore -build"
-    Write-Host ""
-    Write-Host "CoreFx specific actions:"
-    Write-Host "  -buildtests             Build test projects in the solution"
-    Write-Host "  -clean                  Clean the solution"
-    Write-Host ""
-    Write-Host "CoreFx specific options:"
-    Write-Host "  -framework              The target group assemblies are built for (short: -f)"
-    Write-Host "  -os                     The operating system assemblies are built for"
-    Write-Host "  -allconfigurations      Build packages for all build configurations"
-    Write-Host "  -coverage               Collect code coverage when testing"
-    Write-Host "  -outerloop              Include tests which are marked as OuterLoop"
-    Write-Host "  -arch                   The architecture group (x86, x64, arm, etc.)"
-    Write-Host ""
+  Write-Host "Common settings:"
+  Write-Host "  -framework              The target group assemblies are built for (short: -f)"
```

Comment:
Should this include an example or two, as does configuration? e.g. "The garget group assemblies are built for (short: -f), e.g. netcoreapp, uap, or netfx."